packages/create-emotion-styled/types/react.d.ts

```diff
@@ -69,3 +69,11 @@ export interface CreateStyled<Theme extends object = any> {
     options?: StyledOptions,
   ): CreateStyledOtherComponent<IP, Theme>;
 }
+
+export type CreateStyledShorthands<Theme extends object> = {
+  [T in keyof JSX.IntrinsicElements]: CreateStyledOtherComponent<JSX.IntrinsicElements[T], Theme>;
+};
+
+export interface CreateStyled<Theme extends object = any>
```

Comment:
Is there a reason this needs `Theme extends object = any` whereas the shorthand one doesn't need `= any`?

Reply:
@mitchellhamilton `CreateStyled` is access point of all API, where `CreateStyledShorthands` is just a base interface (well, it's not really interface, but in some way, it's interface) of `CreateStyled`.
Therefore, `CreateStyled` is one used by user, so I put the default type on this interface only.